docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+    ... // other unrelated sidebar items
+```
+
+---
+
+2. Create a new top-level external link to the doc
+
+<div className="flex">
+<img src="../img/top-level-external-link.png" alt="top-level external link" style={{ width: '300px', marginRight: '32px' }} />
+
+<div>
+
+If you'd like to add a new top-level external link, <code>sidebars.js</code> requires an object expecting <code>type</code>, <code>label</code>, and <code>href</code> that will look something like this to reflect the screenshot:
+
+```js
+module.exports = {
+  sidebar: [
+     "welcome",
+   {
+     type: "link",
+     label: "My Flying Nimbus",
+     href: "https://magicalyellowcloud.com",
+   },
+   ... // other unrelated sidebar items
+```
+
+You'll want to make sure `type` is set to `"link"`.
+
+</div>
+</div>
+
+---
+
+3. Create a new internal link to the doc in an existing or new category
+
+<div className="flex">
+<div>
+
+In Docusaurusland, sidebar menu items with a down caret are called "categories."
+
+To add your internal page to an existing category, find the category via the `label` item in `sidebars.js` which will match the category text you see displayed in the sidebar (see code below). Find the `items` array for that category and add the doc ID you wish to link internally to.
+
+To add your internal page to a new category, copy and paste the entire object shown below where you'd like it to show up in the sidebar. Edit your label as desired and change `items` to `items: ["my-flying-nimbus"]` with your doc ID.
+
+</div>
+<img src="../img/category-link.png" alt="category link" style={{ width: '300px', marginLeft: '16px' }} />
+</div>
+
+```js
+module.exports = {
+  sidebar: [
+     ... // other unrelated sidebar items
+    {
+      type: "category",
+      label: "Experiment Owners", // what you're looking for, or creating
+      items: ["experiment-owners-root", "my-flying-nimbus"], // doc ID is in this array
+    },
+    ... // other unrelated sidebar items
+```
+
+---
+
+4. Create a new external link to a doc in an existing or new category
+
+<div className="flex">
+<img src="../img/category-external-link.png" alt="category link" style={{ width: '300px', marginRight: '32px', marginBottom: '16px' }} />
+<div>
+
+The code changes for this option essentially combine options 2 and 3.
+
+To add your external link to an existing category, find the category via the `label` item in `sidebars.js` which will match the category text you see displayed in the sidebar (see code below). Find the `items` array for that category and add the object shown below (also shown in option 2) to the array.
+
+To add your external link to a new category, copy and paste the entire object shown below where you'd like it to show up in the sidebar. Edit your top-level label as desired, remove `"experiment-owners-root",` from the `items` array, and update the label you'd like to display for the link.
+
+</div>
+</div>
+
+```js
+module.exports = {
```

Comment:
Optional, but could be useful here and good to know for future code snippets: Docusaurus supports snippet titles: https://v2.docusaurus.io/docs/markdown-features/code-blocks#code-title

(I'll let you decide if this is something we should document, I'm not sure how widely used it'd be)

Reply:
_This is cool_